stable/datadog/templates/agent-clusterchecks-deployment.yaml

```
@@ -65,9 +65,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: spec.nodeName
-{{- if .Values.clusterChecksRunner.env }}
-{{ toYaml .Values.clusterChecksRunner.env | indent 10 }}
-{{- end }}
+        {{- range $index, $value := .Values.clusterChecksRunner.env }}
+          - name: {{ $index }}
+            value: {{ $value | quote }}
+        {{- end }}
```

`env` is an array. The name of the environment variable mustn’t be the index in the array.
```suggestion
        {{- range .Values.clusterChecksRunner.env }}
          - name: {{ .name }}
            value: {{ .value | quote }}
        {{- end }}
```